website/src/css/custom.css

```
@@ -7,7 +7,7 @@
 
 /* You can override the default Infima variables here. */
 :root {
-    --ifm-color-primary: #25c2a0;
+    --ifm-color-primary: #007E60;
```

```suggestion
    --ifm-color-primary: #007e60;
```

I think prettier wants lower-case hex colors, hence the failing build. The comparison looks good to me, there will be a deployment preview published when CI passes.